entc/gen/graph.go

```diff
@@ -598,29 +599,52 @@ func (g *Graph) typ(name string) (*Type, bool) {
 	return nil, false
 }
 
-// templates returns the template.Template for the code and external templates
-// to execute on the Graph object if provided.
+// templates returns the Template to execute on the Graph,
```

Comment:
previosly it was `templates.Template`. Removed on purpose?

Reply:
Yes.